components/connector-service-token-request/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.10.3 as builder
+
+ARG DOCK_PKG_DIR=/go/src/github.com/kyma-project/kyma/components/connector-service-token-request
+
+RUN mkdir -p $DOCK_PKG_DIR
+
+COPY ./ $DOCK_PKG_DIR
+WORKDIR $DOCK_PKG_DIR
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o manager ./cmd/manager
+
+FROM alpine
```

Comment:
we should use fixed versions. In my understanding that image is like "latest"

Reply:
Good catch! I missed that. Fixed.